w3af/plugins/grep/cdn_providers.py

```diff
@@ -0,0 +1,255 @@
+from w3af.core.controllers.plugins.grep_plugin import GrepPlugin
+from w3af.core.data.kb.info import Info
+from w3af.core.data.kb.info_set import InfoSet
+
+
+class cdn_providers(GrepPlugin):
+    """
+    Check CDN (Content Delivery Network) providers used by the website.
+
+    :author: Jakub Peczke (qback.contact@gmail.com)
+    """
+    # CDN headers stored in format ['header-name', 'header-value', 'provider's name']
+    cdn_headers = (
+        ["server", "cloudflare", "Cloudflare"],
```

Comment:
All strings in w3af are single-quote delimited. PEP-8 says:

> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it

So let's stick with single quotes :-)